addons/hr_homeworking/wizard/calendar_popover_delete_wizard_homework.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+
+class CalendarPopoverDeleteWizardHomework(models.TransientModel):
+    _name = 'calendar.popover.delete.wizard.homework'
+    _description = 'Calendar Popover Delete Wizard Homework'
+
+
+    record = fields.Many2one('hr.homeworking')
```

Comment:
Use meaningful variable names
```suggestion
    hr_homeworking_id = fields.Many2one('hr.homeworking')
```